src/test/scala/karazin/scala/users/group/week1/homework/HomeworkSpecification.scala

```diff
@@ -23,15 +23,29 @@ object BooleanOperatorsSpecification extends Properties("Boolean Operators"):
   property("and") = forAll { (pair: (Boolean, Boolean)) =>
     val (left, right) = pair
     
-    and(left, right) == left && right
+    and(left, right) == (left && right)
+  }
+
+  property("lazy and") = forAll { (bool: Boolean) =>

```

Comment:
I'd prefer to throw an exception without a boolean flag. You already checked a happy path. Not needed to overcomplicate this test.

Reply:
Hint please use construction like:
```
  property("simple check") = propBoolean {
    1 == 2
  }
```